examples/006_filter_composite_data_example.py

```diff
@@ -55,16 +58,52 @@
 stress_operator.inputs.bool_rotate_to_global(False)
 stress_field = stress_operator.get_output(pin=0, output_type=dpf.types.fields_container)[0]
 
+# %%
+# Filter data by analysis ply
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# %%
+# List all available analysis plies.
+all_ply_names = get_all_analysis_ply_names(composite_model.get_mesh())
+all_ply_names
+
+# %%
+# The easiest way to filter data by analysis ply is to use the function :func:`.get_ply_wise_data`.
+# This function supports different reduction strategies such as computing the average
+# or maximum/minimum over the spot locations.
+# It also supports selecting a specific spot (TOP, MID, BOT) directly.
+# In this example we select the maximum value over all spots for each node and then request
+# the elemental location which implies averaging over all nodes in an element.
```

Comment:
```suggestion
# This example selects the maximum value over all spots for each node and then requests
# the elemental location, which implies averaging over all nodes in an element.
```
Avoid "we" and either use "you" or make another noun the subject, as I did here by making "example" the subject.